src/templates/_macros/app_tile.html

```diff
@@ -57,6 +60,13 @@
   </{{ tag }}>
 
   {% if is_detail %}
+    {% if app.isWebsite %}
```

Comment:
I briefly considered trying to move it in CSS but I think the button is in a span that is positioned beside the image so it would be weird to try and get it to be under the image.